loadflow-validation/src/main/java/eu/itesla_project/loadflow/validation/ValidationTool.java

```diff
@@ -68,9 +68,15 @@ public Options getOptions() {
                     .desc("verbose output")
                     .build());
             options.addOption(Option.builder().longOpt("output-format")
-                    .desc("output format")
+                    .desc("output format (CSV/CSV_MULTILINE)")
                     .hasArg()
-                    .argName("FLOWS_WRITER")
+                    .argName("VALIDATION_WRITER")
+                    .build());
+            options.addOption(Option.builder().longOpt("type")
+                    .desc("validation type (FLOWS/GENERATORS/...)")
+                    .hasArg()
+                    .argName("VALIDATION_TYPE")
+                    .required()
```

Comment:
The validation, besides the result (ok/nok), produces, as output, a file, with the validated equipments data (file given as input to the command). If we want to be able to run all the validations (flows/generators/etc.) the command should be changed, e.g. providing as input, instead of a file, a folder, where all the files would be stored